src/hotspot/share/utilities/elfFile.cpp

```diff
@@ -34,6 +34,7 @@
 #include "jvm_io.h"
 #include "logging/log.hpp"
 #include "memory/allocation.inline.hpp"
+#include "memory/resourceArea.hpp"
```

Comment:
Shall we only include `resourceArea.hpp` for `defined(PPC64) && !defined(ABI_ELFv2)`?

Reply:
I don't like complicated include guards in include blocks, so I'd prefer not to.

Reply:
I agree with @shipilev on this.